management/templates/system-backup.html

```diff
@@ -235,6 +261,13 @@ <h3>Available backups</h3>
           var host = hostpath.shift();
           $("#backup-target-s3-host").val(host);
           $("#backup-target-s3-path").val(hostpath.join('/'));
+        } else if (r.target.substring(0, 5) == "b2://") {
+          $("#backup-target-type").val("b2");
+          var hostpath = r.target.substring(5).split('@');
```

Comment:
A better name for hostpath would be `bucketPath`.

Some validation of the parts would be useful. I don't see any of the other backends doing it however.
I'm not sure of a good regex to check it...
```
if (usernamepass.length !== 2) {
  alert('warning, format should be b2://username:password@bucket/backuppath');
}
```